### DISCUSSION TITLE
How do you load Trait settings from HTML?

### DISCUSSION THREAD

**Discussion by advancedsoftwarecanada** [GrapesJS/grapesjs#5751] [Q&A]
Have a look at this demo:

https://jsfiddle.net/artur_arseniev/xfvo50hj

When you first see the page... nothing special happens with the type demo.

You must delete the text, then you can add the custom block and configure its traits.

How do you SAVE this data? Right now I am saving this as HTML. 

When I refresh the page, I load the HTML in dynamically. How do I get the trait data?

**Comment by advancedsoftwarecanada** [comment]
OK I think I see the problem... I can't use local storage. I wonder if I can write that local storage to the database? 

Thanks I will investigate.

**Comment by artf** [comment]
Traits should not be stored otherwise updating a component with new traits would be a problem.

**Reply by advancedsoftwarecanada** [reply]
Ahh this explains why I can't find documentation on it! But to your point, I'm already unable to update a component, it seems like its written as HTML. 

This is a blocker for me... I will have to overcome it. Yikes.
